dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -174,13 +271,20 @@ index_metadata(){
         _download_kubeconfig $(_get_cluster_id ${CLUSTER_NAME}) ./kubeconfig
         export KUBECONFIG=./kubeconfig
     fi
+    if [[ $INSTALL_METHOD == "osd" ]]; then
+        export PLATFORM="AWS"
```

Comment:
we need to consider another platform name, as AWS is selected on benchmarks and tests as classic OCP on AWS (non managed). It could be confused when grouping clusters for results

Reply:
good catch, lets do `AWSMS`  AWS Managed-Services ?